FoodBookApp/UI/Views/Browse/BrowseView.swift

```diff
@@ -50,6 +50,17 @@ struct BrowseView: View {
                 
             }
         }
+        .onAppear { // TODO: add this to other view maybe?
```

Comment:
Decided to put it here in this view since at the end of the day the user has to go back to the browse view to update the reviews

Reply:
And one of the latests commits also added it to the spot detail view